flink-libraries/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -354,6 +398,23 @@ private void callSelect(SqlCommandCall cmdCall) {
 		}
 	}
 
+	private boolean callInsertInto(SqlCommandCall cmdCall) {
+		terminal.writer().println(CliStrings.messageInfo(CliStrings.MESSAGE_SUBMITTING_STATEMENT).toAnsi());
+		terminal.flush();
+
+		try {
+			final ProgramTargetDescriptor programTarget = executor.executeUpdate(context, cmdCall.operands[0]);
+			terminal.writer().println(CliStrings.messageInfo(CliStrings.MESSAGE_STATEMENT_SUBMITTED).toAnsi());
```

Comment:
```
programTarget.writeTo(terminal.writer())
```
? It would be easier to add more fields in the future and more difficult to forget about printing them.

Reply:
Initially I had it similar like your proposal, but this would mix a data model class and visualization. `ProgramTargetDescriptor` should not be responsible how it is represented in the CLI.

Reply:
hmmm, what about
```
terminal.writer().println(CliStrings.messageInfo(CliStrings.MESSAGE_STATEMENT_SUBMITTED).toAnsi());
terminal.writer().println(programTarget.toString())
```?